src/channel.rs

```diff
@@ -875,6 +875,13 @@ impl Channel {
 impl Channel {
     /// Construct a `Channel` from a url string.
     ///
+    /// Note: from_url can only be used defining the rss dependency in the Cargo.toml as follows:
```

Comment:
> /// Note: from_url can only be used by enabling the `from_url` feature in Cargo.toml as follows:

I think that would sound better since it explicitly mentions the feature name.